source/app.ts

```diff
@@ -0,0 +1,90 @@
+import { loadCustomConfiguration } from './config'
+import { CpuMonitor } from './monitors/cpu-monitor'
+import { MemoryMonitor } from './monitors/memory-monitor';
+import { NetworkMonitor } from './monitors/network-monitor';
+import { DiskMonitor } from './monitors/disk-monitor';
+
+const config = loadCustomConfiguration()
+const monitors: any[] = [];
+
+function loadMonitors() {
+    const monitorNames = config.monitorNames;
+
+    for (let i = 0; i < monitorNames.length; i++) {
```

Comment:
Here and further: I suppose, it's better to use `monitorNames.forEach(name => ...)`